Vibees/app/src/main/java/com/example/vibees/screens/home/HelpScreen.kt

```diff
@@ -0,0 +1,133 @@
+package com.example.vibees.screens.home
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.toMutableStateList
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun HelpScreen() {
+    Box(
+        modifier = Modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center
+    ) {
+        CollapsableLazyColumn(
+            sections = listOf(
+                CollapsableSection(
+                    title = "About Us",
+                    rows = listOf("Welcome to Vibees, the ultimate party experience app! We are a team of enthusiastic university undergraduates who are passionate about revolutionizing the way parties are hosted and enjoyed. Our goal is to create a platform that empowers individuals to craft personalized and unforgettable events, while prioritizing safety and inclusivity.\n" +
```

Comment:
Strings should be put in the res/values/strings.xml file.  You can then call it using R.values.strings or something similar